cmd/csi-resizer/main.go

```diff
@@ -0,0 +1,85 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"flag"
+	"time"
+
+	"github.com/kubernetes-csi/external-resizer/pkg/controller"
+	"github.com/kubernetes-csi/external-resizer/pkg/resizer"
+	"github.com/kubernetes-csi/external-resizer/pkg/util"
+
+	"k8s.io/klog"
+)
+
+var (
+	master       = flag.String("master", "", "Master URL to build a client config from. Either this or kubeconfig needs to be set if the provisioner is being run out of cluster.")
+	kubeConfig   = flag.String("kubeconfig", "", "Absolute path to the kubeconfig")
+	resyncPeriod = flag.Duration("resync-period", time.Minute*10, "Resync period for cache")
+	workers      = flag.Int("workers", 10, "Concurrency to process multiple resize requests")
```

Comment:
We need a value of controlling csi timeouts. I filed that as a separate issue.

Reply:
Thanks! I will add CSI related implement in next PR.